packages/create-block/lib/templates/controls-esnext/src/index.js.mustache

```diff
@@ -13,19 +13,13 @@ import { registerBlockType } from '@wordpress/blocks';
 import { __ } from '@wordpress/i18n';
 
 /**
- * Lets webpack process CSS, SASS or SCSS files referenced in JavaScript files.
- * All files containing `style` keyword are bundled together. The code used
- * gets applied both to the front of your site and to the editor.
- *
- * @see https://www.npmjs.com/package/@wordpress/scripts#using-css
- */
-import './style.scss';
```

Comment:
@fabiankaegy can we discuss this import during out meet up, i noticed some odd behavior when I left it included, causing my block to NOT show up in the editor, even when plugin is activated.